build.sh

```diff
@@ -51,14 +51,14 @@ check_version () {
 
     if [ $major -ge 4 ]; then
         if [ $minor -eq 0 -a $patch -le 16 ]; then
-            echo "ERROR: version unsupported ($product_version <= 4.0.17)"
+            echo "ERROR: version unsupported ($product_version <= 4.0.16)"
```

Comment:
@krystian-hebel i think `($product_version < 4.0.17)` would be better in terms of readability and understandability.